### PR TITLE
add custom JVM opts

### PR DESCRIPTION
**Pull request by sritchie**
The algebird build was 4x faster with these on travis.

### CODE REVIEW

File: scalding-core/src/main/scala/com/twitter/scalding/Tool.scala [hunk 1]
```diff
@@ -65,10 +65,10 @@ class Tool extends Configured with HTool {
   def run(args: Array[String]): Int = {
     val (mode, jobArgs) = parseModeArgs(args)
     // Connect mode with job Args
-    run(getJob(Mode.putMode(mode, jobArgs)))
+    runJob(getJob(Mode.putMode(mode, jobArgs)))
   }
 
-  protected def run(job: Job): Int = {
```
Comment (johnynek) [inline scalding-core/src/main/scala/com/twitter/scalding/Tool.scala:71]: What's going on here?

Comment (sritchie) [reply scalding-core/src/main/scala/com/twitter/scalding/Tool.scala:71]: oh shit, sorry, this must have been polluting my branch.

**Comment by sritchie** [comment]
Things were not any faster! I wonder if these projects already have `SBT_OPTS` set.

**Comment by piyushnarang** [comment]
@sritchie I've also noticed that travis build have a pretty large variance. So you might want to run a couple of times to make sense of the noise. These settings look reasonable though so we could turn them on anyway.

**Pull request closed without merging**